src/coreclr/jit/gentree.h

```diff
@@ -8039,17 +8039,14 @@ struct GenTreePutArgSplit : public GenTreePutArgStk
 // Represents GT_COPY or GT_RELOAD node
 //
 // As it turns out, these are only needed on targets that happen to have multi-reg returns.
-// However, they are actually needed on any target that has any multi-reg ops. It is just
-// coincidence that those are the same (and there isn't a FEATURE_MULTIREG_OPS).
+// However, they are actually needed on any target that has any multi-reg ops.
```

Comment:
```suggestion
```
Otherwise the comments contradict themselves:
```
... are only needed on targets that happen to have multi-reg returns ...
... are actually needed on any target that has any multi-reg ops ...
```